bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoVacuumHandler.java

```diff
@@ -131,8 +131,9 @@ private boolean updateVacuumStatus(JsonObject statusData) {
         updateState(CHANNEL_CLEAN_TIME,
                 new DecimalType(TimeUnit.SECONDS.toMinutes(statusData.get("clean_time").getAsLong())));
         updateState(CHANNEL_DND_ENABLED, new DecimalType(statusData.get("dnd_enabled").getAsBigDecimal()));
-        updateState(CHANNEL_ERROR_CODE,
-                new StringType(VacuumErrorType.getType(statusData.get("error_code").getAsInt()).getDescription()));
+        VacuumErrorType error_code = VacuumErrorType.getType(statusData.get("error_code").getAsInt());
```

Comment:
Sorry for catching this not in the earlier review, but the codechecker also complaints that this is not camelcase
Should be changed to errorCode